GeneralsMD/Code/GameEngineDevice/Source/W3DDevice/GameClient/W3DView.cpp

```diff
@@ -503,7 +503,8 @@ void W3DView::calcCameraConstraints()
 		center.x -= bottom.x;
 		center.y -= bottom.y;
 
-		Real offset = center.length();
+		// TheSuperHackers @bugfix Mauller 22/10/2025 Halve the camera offset to allow the camera to move closer to the map edges
+		Real offset = center.length() * 0.5f;
```

Comment:
I do not yet agree with this change, not before addressing issue #1270, because #1270 is responsible for being able to look further beyond the border. So better first fix the border boundary, then tweak if required.